src/main/webapp/WEB-INF/jsp/tiles/views/home.jsp

```diff
@@ -16,6 +16,11 @@
 <div class="row column margin-bottom-xlarge expanded">
   <tiles:insertAttribute name="experiments-summary-panel"/>
 </div>
+<div class="row expanded margin-top-large" data-equalizer>
```

Comment:
Since we don’t have several boxes that need to have equal height we can remove `data-equalizer`. Read more about it at https://foundation.zurb.com/sites/docs/equalizer.html.

Also, verify that elements are exactly the same width. I suspect that unless you use `row column`, nesting `small-12 columns` below will add some unwanted extra margin.

Reply:
I have deleted `data-equalizer` and tested `row column` and `small-12 columns` issue. The current version keeps the same width as the other panels without extra margin.